test/parallel/sdk.API.utils.test.ts

```diff
@@ -79,7 +80,51 @@ beforeEach(async () => {
   );
 });
 
-test("isBuyAssetTransactionSuccessful returns a true flag", async () => {
+test("GIVEN buyAsset WHEN operation is confirmed AND isBuyAssetTransactionSuccessful THEN it returns true", async () => {
+  const multiSwapOutput = await signSendFinalized(
```

Comment:
this is not a multiswap operation. 
xyk.buyAsset and xyk.sellAsset static methods runs sell and by operations.  Am I right?

Reply:
changed